apps/src/templates/teacherNavigation/ResourceRow.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import {BodyTwoText, StrongText} from '@cdo/apps/componentLibrary/typography';
+
+import styles from './lesson-materials.module.scss';
+
+export type Lesson = {
+  name: string;
+  id: number;
+};
+
+type ResourceRowProps = {
+  lessonNumber: number | null;
+  unitNumber: number | null;
+  resource: {
+    key: string;
+    name: string;
+    url: string;
+    downloadUrl: string | null;
+    audience: string;
+    type: string;
+  } | null;
```

Comment:
If we make sure only to render this component when there's actually a resource there, we don't need to let any of the props be null, and that makes life easier downstream.

```suggestion
  lessonNumber: number;
  unitNumber: number;
  resource: {
    key: string;
    name: string;
    url: string;
    downloadUrl: string | null;
    audience: string;
    type: string;
  };
```